gitlab/resource_gitlab_service_microsoft_teams.go

```diff
@@ -0,0 +1,228 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabServiceMicrosoftTeams() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabServiceMicrosoftTeamsCreate,
+		Read:   resourceGitlabServiceMicrosoftTeamsRead,
+		Update: resourceGitlabServiceMicrosoftTeamsUpdate,
+		Delete: resourceGitlabServiceMicrosoftTeamsDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceGitlabServiceMicrosoftTeamsImportState,
```

Comment:
This resource can use [schema.ImportStatePassthrough](https://www.terraform.io/plugin/sdkv2/resources/import#importer-state-function) which is preferred. You would need to update your *Read function to parse the ID instead of getting the project attribute.